alien4cloud-yorc-plugin/src/main/java/org/ystia/yorc/alien4cloud/plugin/YstiaOrchestratorFactory.java

```diff
@@ -47,11 +47,14 @@
 
     @Override
     public YorcPaaSProvider newInstance() {
-        return beanFactory.getBean(YorcPaaSProvider.class);
+        YorcPaaSProvider instance = beanFactory.getBean(YorcPaaSProvider.class);
+        instance.startLogsAndEvents();
+        return instance;
     }
 
     @Override
     public void destroy(YorcPaaSProvider instance) {
```

Comment:
Is this method called when disabling an orchestrator or when destroying it?

Reply:
On disabling: it is used in the `disable` method in the alien core code: https://github.com/alien4cloud/alien4cloud/blob/develop/alien4cloud-core/src/main/java/alien4cloud/orchestrators/services/OrchestratorStateService.java#L238